<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>terminal</class>
 <widget class="QMainWindow" name="terminal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>terminal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
border:10px solid gray;
background-color: rgb(0, 0, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>122</y>
      <width>40</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color: qlineargradient(spread:pad, x1:0.489, y1:0.0170455, x2:0.499789, y2:0.972, stop:0 rgba(255, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
border:1px solid yellow;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>192</y>
      <width>40</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color: rgb(0, 255, 255);
border:1px solid yellow;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>148</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border-image: url(:/images/arrow4.png);
border:0;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_GPRS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font:bold 10pt;
background-color:black;
color:white;
border-top:1px transparent;
border-left:1px transparent;
border-right:1px solid yellow;
border-bottom:1px transparent;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;GPRS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="gprs_state">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>41</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:black;
color:white;
border-top:1px transparent;
border-left:1px transparent;
border-right:1px solid yellow;
border-bottom:1px solid yellow;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;T&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_GPS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>73</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font:bold 10pt;
background-color:black;
color:white;
border-top:1px transparent;
border-left:1px transparent;
border-right:1px solid yellow;
border-bottom:1px transparent;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;GPS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="gps_state">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>104</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:black;
color:white;
border-top:1px transparent;
border-left:1px transparent;
border-right:1px solid yellow;
border-bottom:1px solid yellow;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;T&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>136</y>
      <width>90</width>
      <height>63</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font:bold 10pt;
background-color:black;
color:white;
border-top:1px transparent;
border-left:1px transparent;
border-right:1px solid yellow;
border-bottom:1px solid yellow;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Message&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Record">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>199</y>
      <width>90</width>
      <height>63</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font:bold 10pt;
background-color:black;
color:white;
border-top:1px transparent;
border-left:1px transparent;
border-right:1px solid yellow;
border-bottom:1px solid yellow;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Record&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_start">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font:bold 10pt;
background-color:black;
color:white;
border-top:1px transparent;
border-left:1px transparent;
border-right:1px solid yellow;
border-bottom:1px transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_terminal">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>41</y>
      <width>120</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font:bold 10pt;
background-color:black;
color:white;
border-top:1px solid yellow;
border-left:1px transparent;
border-right:1px solid yellow;
border-bottom:1px solid yellow;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bus_number">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>41</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font:bold 10pt;
background-color:black;
color:white;
border-top:1px solid yellow;
border-left:1px transparent;
border-right:1px transparent;
border-bottom:1px solid yellow;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="speed">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>93</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font:bold 10pt;
background-color:black;
color:white;
border-top:1px transparent;
border-left:1px solid yellow;
border-right:1px transparent;
border-bottom:1px solid yellow;
}</string>
    </property>
    <property name="text">
     <string>km/h</string>
    </property>
   </widget>
   <widget class="QLabel" name="limit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>93</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font:bold 10pt;
background-color:black;
color:white;
border-top:1px transparent;
border-left:1px transparent;
border-right:1px transparent;
border-bottom:1px solid yellow;
}</string>
    </property>
    <property name="text">
     <string>km/h</string>
    </property>
   </widget>
   <widget class="QLabel" name="cur_station">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>73</y>
      <width>130</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font:bold 14pt;
color:yellow;
border:1px transparent;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TextLabel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="next_station">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>212</y>
      <width>130</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font:bold 14pt;
color:yellow;
border:1px transparent;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TextLabel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_speed_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>93</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font:bold 7pt;
background-color:black;
color:white;
border-top:1px transparent;
border-left:1px transparent;
border-right:1px solid yellow;
border-bottom:1px solid yellow;
}</string>
    </property>
    <property name="text">
     <string>km/h</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_limit_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>93</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font:bold 7pt;
background-color:black;
color:white;
border-top:1px transparent;
border-left:1px transparent;
border-right:1px transparent;
border-bottom:1px solid yellow;
}</string>
    </property>
    <property name="text">
     <string>km/h</string>
    </property>
   </widget>
   <widget class="QLabel" name="BUS">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>41</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font:bold 10pt;
background-color:black;
color:white;
border-top:1px solid yellow;
border-left:1px transparent;
border-right:1px transparent;
border-bottom:1px solid yellow;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;BUS&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Start">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font:bold 10pt;
background-color:black;
color:white;
border-top:1px transparent;
border-left:1px transparent;
border-right:1px transparent;
border-bottom:1px transparent;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Start:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Terminal">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>41</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font:bold 10pt;
background-color:black;
color:white;
border-top:1px solid yellow;
border-left:1px transparent;
border-right:1px transparent;
border-bottom:1px solid yellow;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Terminal:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_TDD">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>232</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:white;
border:1px transparent;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TDD&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Speed">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>73</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font:bold 10pt;
background-color:black;
color:white;
border-top:1px transparent;
border-left:1px solid yellow;
border-right:1px solid yellow;
border-bottom:1px transparent;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Speed&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Limit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>73</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font:bold 10pt;
background-color:black;
color:red;
border-top:1px transparent;
border-left:1px transparent;
border-right:1px transparent;
border-bottom:1px transparent;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Limit&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>170</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font:bold 10pt;
background-color:black;
color:white;
border-top:1px transparent;
border-left:1px transparent;
border-right:1px transparent;
border-bottom:1px transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
